packages/adminrouter/extra/src/test-harness/modules/generic_test_code/generalised_tests.py

```diff
@@ -61,15 +61,8 @@ def _verify_endpoint_tests_conf(endpoint_tests):
         _check_all_keys_are_present_in_dict(t, ['tests', 'type'])
         _verify_type_specification(t['type'])
 
-        at_least_one_test_enabled = False
-        assert 0 < len(t['tests'].keys()) < 7
-        for k in t['tests']:
-            assert 'enabled' in t['tests'][k]
-            assert t['tests'][k]['enabled'] in [True, False]
-            at_least_one_test_enabled = at_least_one_test_enabled or \
-                t['tests'][k]['enabled']
-
-        assert at_least_one_test_enabled
+        # At least one test is there and not more than 7:
```

Comment:
Please comment the criteria for changing this number as I can imagine this verification breaking and the author being confused.